docs/04-guides/on-demand/01-upload-video-asset/02-api.mdx

```diff
@@ -208,3 +210,101 @@ curl --location --request GET 'https://livepeer.studio/api/asset/{id}' \
 
 </CodeBlockTabItem>
 </CodeBlockTabs>
+
+## Resumable Upload
+
+The resumable upload method allows a user to resume their upload if their 
+connection fails due to a cancelled request or a dropped connection. If you're
+uploading a large file and you lose connection at 98%, you don't have to restart
+from the beginning, you can continue back at 98%.
+
+### Step 1: Generate upload URL
+
+To upload the asset to the Livepeer network locally, you'll need to make a POST
+request to generate an upload URL.
+
+<CodeBlockTabs>
+<CodeBlockTabItem value="JavaScript" label="JavaScript">
+
+```js
+const uploadAsset = await fetch(
```

Comment:
Let's refrain from naming these `const` variables using verbs. Verbs suggest the `const` is a function but this fetch call does not return a function.

`uploadAsset`, `getAsset`, etc should be renamed with a noun.

Reply:
Example
`const uploadAsset` should be `const { url }`

Reply:
Can you expand on this point a bit more? Is the reason why the `const` is not a function because it doesn't return anything and is just a fetch call?

Reply:
The response from most of these `fetch` calls don't return a function they return an object. functions names == verbs, object names = nouns